app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/terminationdate/TerminationDateDestination.kt

```diff
@@ -49,100 +50,178 @@ import kotlinx.datetime.toLocalDateTime
-          .verticalScroll(rememberScrollState())
-          .windowInsetsPadding(WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal)),
-      ) {
-        val sideSpacingModifier = if (windowSizeClass.widthSizeClass == WindowWidthSizeClass.Expanded) {
-          Modifier
-            .fillMaxWidth(0.8f)
-            .wrapContentWidth(Alignment.Start)
-            .align(Alignment.CenterHorizontally)
-        } else {
-          Modifier.padding(horizontal = 16.dp)
-        }
-        Spacer(Modifier.height(20.dp))
-        ChatCard(sideSpacingModifier)
-        Spacer(Modifier.height(20.dp))
-        Spacer(Modifier.weight(1f))
-        DatePickerCard(
-          datePickerState = uiState.datePickerState,
-          modifier = sideSpacingModifier,
-        )
-        Spacer(Modifier.height(16.dp))
-        HedvigContainedButton(
-          text = stringResource(hedvig.resources.R.string.general_continue_button),
-          onClick = submit,
-          enabled = uiState.canSubmit,
-          modifier = sideSpacingModifier,
-        )
-        Spacer(Modifier.height(16.dp))
-        Spacer(
-          Modifier.windowInsetsPadding(
-            WindowInsets.safeDrawing.only(WindowInsetsSides.Bottom),
-          ),
-        )
-      }
     }
-    HedvigFullScreenCenterAlignedProgressDebounced(show = uiState.isLoading)
+    Spacer(Modifier.height(16.dp))
+    HedvigContainedButton(
+      text = stringResource(id = R.string.TERMINATION_FLOW_CANCEL_INSURANCE_BUTTON),
+      onClick = submit,
+      enabled = uiState.canSubmit,
+      modifier = Modifier.padding(horizontal = 16.dp),
+    )
+    Spacer(Modifier.height(16.dp))
   }
 }
 
 @Composable
-private fun ChatCard(modifier: Modifier = Modifier) {
+private fun ImportantInfoCheckBox(isChecked: Boolean, onCheckedChange: () -> Unit, modifier: Modifier = Modifier) {
   HedvigCard(
-    shape = RoundedCornerShape(12.dp),
-    elevation = HedvigCardElevation.Elevated(),
-    modifier = modifier.padding(end = 16.dp),
+    onClick = null,
+    modifier = modifier,
   ) {
-    Text(
-      text = stringResource(hedvig.resources.R.string.SET_TERMINATION_DATE_TEXT),
-      modifier = Modifier.padding(vertical = 12.dp, horizontal = 16.dp),
-    )
+    Row(
+      verticalAlignment = Alignment.CenterVertically,
+      modifier = Modifier
+        .fillMaxWidth()
+        .padding(horizontal = 16.dp, vertical = 10.dp),
+    ) {
+      Column(modifier = Modifier.weight(1f)) {
+        Text(
+          text = stringResource(id = R.string.TERMINATION_FLOW_IMPORTANT_INFORMATION_TITLE),
+          style = MaterialTheme.typography.bodyLarge,
+        )
+        Text(
+          text = stringResource(id = R.string.TERMINATION_FLOW_IMPORTANT_INFORMATION_TEXT),
+          style = MaterialTheme.typography.bodyMedium,
+          color = MaterialTheme.colorScheme.onSurfaceVariant,
+        )
+        Spacer(Modifier.height(16.dp))
+        HedvigContainedButton(
+          onClick = {},
```

Comment:
I would actually like to have `onCheckedChange()` here as well. Imo it makes sense that the button controls the checkbox, since the checkbox is "inside" the button.